src/components/Calculator.jsx

```diff
@@ -0,0 +1,145 @@
+import { useState } from "preact/hooks";
+
+export default function Calculator() {
+  const [developersCount, setDevelopersCount] = useState(5);
```

Comment:
Let's put a higher number of developers – let's say 20?

Reply:
Good idea. I'll update it.